### PR TITLE
disable resetting colormap values

### PR DESCRIPTION
**Pull request by Bechir-Brahem**
keep the range selected for every new acquisition

### CODE REVIEW

File: CtbGui [hunk 1]
```diff
@@ -2085,12 +2092,18 @@ class MainWindow(QtWidgets.QMainWindow):
...
+                        self.plotAnalogImage.getHistogramWidget().item.setHistogramRange(*histRange,padding=0)
                     
                     
                 # transceiver
                 if self.romode.value in [3, 4]:  
+                    # get zoom state
                     viewBox = self.plotTransceiverImage.getView()
                     state = viewBox.getState()
+                    # get histogram (colorbar) levels and histogram zoom range
+                    levels = self.plotAnalogImage.getHistogramWidget().item.getLevels()
```
Comment (thattil) [inline CtbGui:2105]: Typo probably? You would have to use plotTransceiverImage here

File: CtbGui [hunk 2]
```diff
@@ -2085,12 +2092,18 @@ class MainWindow(QtWidgets.QMainWindow):
...
+                    histRange = self.plotAnalogImage.getHistogramWidget().item.getHistogramRange()
```
Comment (thattil) [inline CtbGui:2106]: same here. Maybe try with the virtual ctb server for moench04 and matterhorn

File: CtbGui [hunk 3]
```diff
@@ -2118,7 +2131,8 @@ class MainWindow(QtWidgets.QMainWindow):
                         self.firstTransceiverImage = False
                     else:
                         viewBox.setState(state)
-
+                        self.plotAnalogImage.getHistogramWidget().item.setLevels(min=levels[0],max=levels[1])
```
Comment (thattil) [inline CtbGui:2134]: same here

File: CtbGui [hunk 4]
```diff
@@ -2140,6 +2154,7 @@ class MainWindow(QtWidgets.QMainWindow):
             + "} :disabled {background-color: grey}")
 
     def showPalette(self, button):
+        print('palette')
```
Comment (thattil) [inline CtbGui:2157]: Mayeb we could lose this print. Assuming it was to test :)

File: CtbGui.ui [hunk 5]
```diff
@@ -92,7 +92,7 @@
             <height>0</height>
            </size>
           </property>
-          <property name="maximumSize">
+          <property name="maximumSize">('init')
```
Comment (thattil) [inline CtbGui.ui:95]: Not sure about this 'init'

**Review by thattil — CHANGES_REQUESTED** [review]

**Comment by Bechir-Brahem** [comment]
fixed the typos for transceiver mode and the others

**Review by thattil — APPROVED** [review]
LGTM

**Pull request merged**